jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/Attributes.java

```diff
@@ -170,60 +170,60 @@ static <T extends Attributes.Wrapper> T unwrap(Attributes attributes, Class<T> t
      */
     class Wrapper implements Attributes
     {
-        protected final Attributes _attributes;
+        private final Attributes _wrapped;
 
-        public Wrapper(Attributes attributes)
+        public Wrapper(Attributes wrapped)
         {
-            _attributes = attributes;
+            _wrapped = wrapped;
         }
 
         public Attributes getWrapped()
         {
-            return _attributes;
+            return _wrapped;
         }
 
         @Override
         public Object removeAttribute(String name)
         {
-            return _attributes.removeAttribute(name);
+            return _wrapped.removeAttribute(name);
```

Comment:
Like in all other wrapper classes, these should call `getWrapped()`, so that a subclass may just override `getWrapped()` instead of all methods.